app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -1,2 +1,13 @@
+$primary: #1a454a;
+$secondary: rgb(241, 240, 240);
```

Comment:
- Is Overwriting the Bootstrap the bootstrap default themes essential?
- Isn't better to use a consistent specification for colors, one of  the `RGB values, HEX values, HSL values, RGBA values, and HSLA values` [more](https://www.w3schools.com/css/css_colors.asp)?
-  If overwriting the variants is essential we can fallow the official recommendation to do it [click here](https://react-bootstrap.netlify.app/getting-started/introduction/#customize-bootstrap).